ivy/functional/backends/numpy/experimental/linear_algebra.py

```diff
@@ -108,3 +108,10 @@ def eig(x: np.ndarray, /) -> Tuple[np.ndarray]:
 
 
 eig.support_native_out = False
+
+
+def eivalsg(x: np.ndarray, /) -> np.ndarray:
```

Comment:
Can you please let me know what is `eivalsg` function here? What is the difference from `eivals`?

Reply:
It is a typo. Thanks for pointing out. Will fix it.